app/src/main/java/com/kickstarter/viewmodels/NotificationsViewModel.kt

```diff
@@ -196,6 +218,8 @@ interface NotificationsViewModel {
             this.userInput.onNext(this.userOutput.value.toBuilder().notifyOfUpdates(checked).build())
         }
 
+        override fun creatorDigestFrequencyIsGone(): Observable<Boolean> = this.creatorDigestFrequencyIsGone
```

Comment:
Little nit but nothing big you can just remove the return type since it's a `single expression function`